test/unit/spec/narrative_core/upload/fileUploadWidget-spec.js

```diff
@@ -211,30 +183,19 @@ define([
             });
         });
 
-        it('Should contain a cancel warning button', () => {
-            $targetNode = $('<div>');
-            let uploadWidget = new FileUploadWidget($targetNode, {
-                path: '/',
-                userInfo: {
-                    user: fakeUser,
-                    globusLinked: false,
-                },
-            });
-
-            // Create file
-            const filename = 'foo.txt';
-            mockUploadEndpoint(filename, fakeUser, false);
-            var mockFile = createMockFile(filename);
+        it('Should contain a cancel warning button', function (done) {
+            document.body.appendChild(this.node);
 
-            const adderMock = jasmine.createSpy('adderMock');
-            uploadWidget.dropzone.on('addedfile', () => {
-                adderMock();
+            this.fuWidget.dropzone.on('sending', () => {
+                const $cancelButton = this.fuWidget.$elem.find('.cancel');
+                expect($cancelButton).toBeDefined();
+                expect($cancelButton.attr('data-dz-remove')).toBeDefined();
+                // prevent the XHR from being submitted -- for some reason jasmine.ajax does not catch it
+                this.fuWidget.dropzone.disable();
```

Comment:
This is a rather ugly fix for the issue, but for some reason the jasmine mock was not working, and there's no way to hack into the dropzone widget to mock out the XHR. This just kills the widget before it can make the XHR call.